piperider_cli/event/__init__.py

```diff
@@ -127,3 +129,12 @@ def log_event(prop, event_type, **kwargs):
         **prop,
     )
     _collector.log_event(payload, event_type)
+
+
+class UserProfileConnector(object):
```

Comment:
just a minor question, why do we name it `Connector`?

Reply:
Rename it to `UserProfileCongigurator`